docs/roman/datamodels/metadata.rst

```diff
@@ -8,74 +8,44 @@ Metadata information associated with a data model is accessed through
 its `meta` member.  For example, to access the date that an
 observation was made::
 
-    print(model.meta.observation.date)
+    print(model.meta.observation.start_time)
 
 Metadata values are automatically type-checked against the schema when
 they are set. Therefore, setting a attribute which expects a number to a
-string will raise an exception::
-
-.. doctest-skip::
-
-    >>> from romancal.datamodels import ImageModel
-    >>> model = ImageModel()
-    >>> model.meta.target.ra = "foo"    # doctest: +SKIP
-    Traceback (most recent call last):
-      File "<stdin>", line 1, in <module>
-      File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
-        object.__setattr__(self, attr, val)
-      File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
-        val = self.to_basic_type(val)
-      File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
-        raise ValueError(e.message)
-    ValueError: 'foo' is not of type u'number'
+string will raise an exception.
+
+.. code-block:: python
+
+        from romancal.datamodels import ImageModel
+        model = ImageModel()
+        model.meta.target.ra = "foo"
+        Traceback (most recent call last):
+          File "<stdin>", line 1, in <module>
+          File "site-packages/romancal.datamodels/schema.py", line 672, in __setattr__
+            object.__setattr__(self, attr, val)
+          File "site-packages/romancal.datamodels/schema.py", line 490, in __set__
+            val = self.to_basic_type(val)
+          File "site-packages/romancal.datamodels/schema.py", line 422, in to_basic_type
+            raise ValueError(e.message)
+        ValueError: 'foo' is not of type u'number'
```

Comment:
This is a truly ancient traceback, doesn't the `u'number'` imply Python 2.x?  Here's the modern error:

```python
ValidationError: While validating ra the following error occurred:
'foo' is not of type 'number'

Failed validating 'type' in schema:
    {'$schema': 'http://stsci.edu/schemas/asdf-schema/0.1.0/asdf-schema',
     'archive_catalog': {'datatype': 'float',
                         'destination': ['ScienceCommon.ra']},
     'sdf': {'source': {'origin': 'PSS:fixed_target.ra_computed'},
             'special_processing': 'VALUE_REQUIRED'},
     'title': 'Target RA at mid time of exposure',
     'type': 'number'}

On instance:
    'foo'
```